diff --git a/gsf/gsf-utils.c b/gsf/gsf-utils.c
index d7a7247..cc52d99 100644
--- a/gsf/gsf-utils.c
+++ b/gsf/gsf-utils.c
@@ -115,9 +115,9 @@ static GTypeModule *static_type_module = NULL;
 #include <windows.h>
 static HMODULE gsf_dll_hmodule;
 BOOL WINAPI
-DllMain (HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved);
+static _disabled_DllMain (HINSTANCE hinstDLL, DWORD fdwReason, LPVOID lpvReserved);
 BOOL WINAPI
-DllMain (HINSTANCE hinstDLL, DWORD fdwReason, G_GNUC_UNUSED LPVOID lpvReserved)
+static _disabled_DllMain (HINSTANCE hinstDLL, DWORD fdwReason, G_GNUC_UNUSED LPVOID lpvReserved)
 {
 	if (fdwReason == DLL_PROCESS_ATTACH) gsf_dll_hmodule = hinstDLL;
 	return TRUE;
-- 
1.7.1

